WordPress/Classes/Utility/Analytics/WPAnalyticsEvent.swift

```diff
@@ -464,6 +474,12 @@ extension WPAnalytics {
         var mergedProperties: [AnyHashable: Any] = event.defaultProperties ?? [:]
         mergedProperties.merge(properties) { (_, new) in new }
 
+        if properties.count == 0 {
+            DDLogInfo("🔵 Tracked: \(event.value)")
+        } else {
+            let propertiesDescription = properties.map({ "\($0.key): \($0.value)"}).sorted().joined(separator: ", ")
+            DDLogInfo("🔵 Tracked:\(event.value) <\(propertiesDescription)>")
+        }
```

Comment:
This ports the console log functionality that's in `WPAnalyticsTrackerAutomatticTracks` to this newer endpoint.

Reply:
I don't know if this is expected, but I'm seeing two console logs for each event (I verified on tracks that each event is only being tracked once though).

Reply:
I'm going to back out this part of the change to figure out what's going on with this then. I wonder if I had a build variable out of sync or something.